kong/consumer_group_test.go

```diff
@@ -242,3 +242,106 @@ func compareConsumerGroups(expected, actual []*ConsumerGroup) bool {
 
 	return (compareSlices(expectedNames, actualNames))
 }
+
+func TestConsumerGroupGetEndpointPreGW39(t *testing.T) {
+	RunWhenEnterprise(t, ">=2.7.0 <3.9.0", RequiredFeatures{})
+	assert := assert.New(t)
+
+	client, err := NewTestClient(nil, nil)
+	require.NoError(t, err)
```

Comment:
Nothing major but this is inconsistent here: we create an `assert` object but for `require` we use the top level funcs from the package.